### PR TITLE
Finish version 63.1, 64, 65.1, 66.1, 67.1, 67.2 patch process, and fix some problem

### PR DESCRIPTION
**Pull request by elijahtsai**
Fixes duraki/SketchCrapp#2
Version 63.1 is now supported.
Change the structure from version base to application base.
Each part of code is addressed along with commit message.

### CODE REVIEW

File: sketchcrapp.sh [hunk 1]
```diff
@@ -130,120 +135,187 @@ clean() {
...
-
-## > Missing openssl library
+## > Check if missing openssl library
 if ! command -v openssl &> /dev/null; then
-  echo "OpenSSL is not installed on your system. This should not happen, macOS have openssl built-in."
+  echo "OpenSSL is not installed on your system."
+  echo "This should not happen, macOS have openssl built-in."
   echo "[FIX] Try: brew install openssl"
+  echo "[FIX] Try: port install openssl"
   echo "[FIX] Try: install openssl manually"
```
Comment (duraki) [inline sketchcrapp.sh:280]: We should exit() here.

File: sketchcrapp.sh [hunk 2]
```diff
@@ -130,120 +135,187 @@ clean() {
...
+  if [ -d "/Applications/Sketch.app" ]; then
+    # /Application
+    echo "[+] Selected Sketch.app path is </Applications> (auto-detected) ... OK"
+    verifyApplication "/Applications/Sketch.app"
+  elif [ -d "$HOME/Applications/Sketch.app" ]; then
+    # ~/Application
+    echo "[+] Selected Sketch.app path is <$HOME/Applications> (auto-detected) ... OK"
+    verifyApplication "$HOME/Applications/Sketch.app"
+  else 
+    echo "Application not found either in /Applications or ~/Applications"
```
Comment (duraki) [inline sketchcrapp.sh:297]: Also, maybe below:

`echo "Try: ./sketchcrapp -a /Custom/Path/For/Applications/Sketch.app"`

File: sketchcrapp.sh (hunk 1 above)
Comment (elijahtsai) [reply sketchcrapp.sh:278]: finish in 4fcadbea796473356c258aada8608c31f59fdab8

File: sketchcrapp.sh (hunk 2 above)
Comment (elijahtsai) [reply sketchcrapp.sh:296]: Finished request in adc7097ca9a65df5639cc8e7bba7c097653cf6c4

**Pull request merged**